### PR TITLE
[Tertiary btn] Remove padding for tertiary button

### PR DESCRIPTION
**Pull request by JulienVannier66**
### What does this PR do?

Remove padding for tertiary skin button

Related to: #

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [ ] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: app/styles/atoms/button.less [hunk 1]
```diff
@@ -91,7 +92,8 @@
 
   transition: all 0.25s ease-in-out;
 
-  i, span {
+  i,
+  span {
```
Comment (Miexil) [inline app/styles/atoms/button.less:96]: Is this prettier's doing again ? 😮‍💨

**Review by Miexil — COMMENTED** [review]
I think you already explained last friday but I forgot.
What's the point of doing this ?
We already have an OSS::Link component

File: app/styles/atoms/button.less (hunk 1 above)
Comment (JulienVannier66) [reply app/styles/atoms/button.less:96]: Yes!

**Comment by JulienVannier66** [comment]
The goal is to have the same design for OSS::Button tertiary and OSS::Link, it is not the case currently. It is help us for future change on it. The usage between these two elements is tech only, maybe we can deprecate one of these in the future without layout update

**Review by phndiaye — APPROVED** [review]

**Review by phndiaye — APPROVED** [review]

**Review by Miexil — APPROVED** [review]

**Pull request merged**